core/tauri-macros/src/lib.rs

```diff
@@ -60,3 +60,44 @@ pub fn generate_context(items: TokenStream) -> TokenStream {
   let path = parse_macro_input!(items as ContextItems);
   context::generate_context(path).into()
 }
+
+/// Adds a default value `crate::Wry` to the last generic item of a struct/enum
+#[doc(hidden)]
+#[proc_macro_attribute]
+pub fn default_runtime_wry(_attrs: TokenStream, input: TokenStream) -> TokenStream {
```

Comment:
Maybe just name it `default_runtime` since it might be another runtime and we'll set it depends on the feature flag in the future.

Reply:
Yeah, and then accept the default runtime in the attributes. will look like `#[default_runtime(Wry)]` or `#[default_runtime(crate::Wry)]` depending on if it was imported (which is probably won't since it's behind a feature flag).